cv/index.html

```diff
@@ -60,37 +60,37 @@
                 <p class="position"><b>Zola</b> / Senior Software Developer</p>
                 <p class="position-duration">APRIL 2022 - OCTOBER 2023, REMOTE</p>
                 <ul class="position-description-list">
-                    <li>Developed new product features using Dropwizard (Java) such as spam reporting, ads in search results, features for marketing to vendors</li>
```

Comment:
here and plusgrade: not 100% sure about dropping or keeping the language & framework. it really puts distance between the verb ("developed") and the examples ("spam reporting, ...")